apps/docs/src/app/platform/component-docs/platform-forms/platform-date-picker/platform-date-picker-examples/platform-date-picker-example.component.html

```diff
@@ -24,7 +24,7 @@ <h3>Date Picker with Reactive Form.</h3>
             [formControl]="ffl4.formControl">
         </fdp-date-picker>
     </fdp-form-field>
-    <fdp-form-field #ffl5 id="disableddate" column="1" rank="2" label="Disabled Date Picker:"
+    <fdp-form-field #ffl5 id="disableddate" column="1" rank="2" required="true" label="Disabled Date Picker:"
```

Comment:
It is better to simply use `required` without `="true"` or `="false"` - the presence of the required property will always evaluate to true: https://stackoverflow.com/a/48651305/2217983 

```suggestion
    <fdp-form-field #ffl5 id="disableddate" column="1" rank="2" required label="Disabled Date Picker:"
```